src/vue/modules/issuance-explorer/wrappers/issuance.js

```diff
@@ -1,8 +1,9 @@
 import _get from 'lodash/get'
+import { Request } from '@modules/requests/shared/wrappers/request'
```

Comment:
The same thing about using other modules components https://github.com/tokend/web-client/pull/229#discussion_r274847633. It concerns the wrappers too.

Also, we should not forget to write unit tests after adding some new features.